products/compute/terraform.yaml

```diff
@@ -1403,6 +1403,35 @@ overrides: !ruby/object:Overrides::ResourceOverrides
     custom_code: !ruby/object:Provider::Terraform::CustomCode
       constants: 'templates/terraform/constants/router_nat.go.erb'
       resource_definition: 'templates/terraform/resource_definition/router_nat.go.erb'
+  RouterBgpPeer: !ruby/object:Overrides::Terraform::ResourceOverride
+    legacy_name: 'google_compute_router_peer'
+    exclude_validator: true
+    id_format: "{{project}}/{{region}}/{{router}}/{{name}}"
```

Comment:
This modifies the current import format of `{{region}}/{{router}}/{{name}}`. We should update this to `projects/{{project}}/regions/{{region}}/routers/{{router}}/{{name}}` in 3.0.0 as part of the id format updates to a standard form

Reply:
Should be back as `{{region}}/{{router}}/{{name}}` for now. We can update to the new format in 3.0.0.